web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html

```diff
@@ -17,15 +17,15 @@
     <div class="label">{{config.fileTypes.label}}</div>
     <mat-form-field layout-fill appearance="outline">
       <mat-select
-        id="file_type_select"
+        id="source-type-select"
```

Comment:
should we be consistent with the label name? source-file-type-select is a better id. It's easier to identify as page object